syntax = "proto3";

package terragen;


message ChunkTransmitModel {

  string id = 1;

  bytes data = 2;
}